src/payment/v2/payment-mappers/payment-intent-mapper.js

```diff
@@ -0,0 +1,73 @@
+import { omitNil } from '../../../common/utils';
+import GatewayMapper from './gateway-mapper';
+import QuoteMapper from './quote-mapper';
+import StoreMapper from './store-mapper';
+
+export default class PaymentIntentMapper {
+    /**
+     * @returns {PaymentIntentMapper}
+     */
+    static create() {
+        const gatewayMapper = GatewayMapper.create();
+        const quoteMapper = QuoteMapper.create();
+        const storeMapper = StoreMapper.create();
+
+        return new PaymentIntentMapper(gatewayMapper, quoteMapper, storeMapper);
+    }
+
+    /**
+     * @param {GatewayMapper} gatewayMapper
+     * @param {QuoteMapper} quoteMapper
+     * @param {StoreMapper} storeMapper
+     */
+    constructor(gatewayMapper, quoteMapper, storeMapper) {
```

Comment:
You're passing `gatewayMapper`, `quoteMapper`, and `storeMapper`. I only see the `gatewayMapper` being used. Are the others necessary? Am I missing something?